src/serializers/graph_extensions.c

```
@@ -129,84 +96,84 @@ void Serializer_Graph_SetNodeLabels
-	GrB_Info info;
-	Delta_Matrix M      = Graph_GetRelationMatrix(g, r, false);
-	Delta_Matrix adj    = Graph_GetAdjacencyMatrix(g, false);
-	GrB_Matrix   m      = Delta_Matrix_M(M);
-	GrB_Matrix   tm     = Delta_Matrix_M(Delta_Matrix_getTranspose(M));
-	GrB_Matrix   adj_m  = Delta_Matrix_M(adj);
-	GrB_Matrix   adj_tm = Delta_Matrix_M(Delta_Matrix_getTranspose(adj));
+	// validations
+	ASSERT(n      >  0);
+	ASSERT(g      != NULL);
+	ASSERT(ids    != NULL);
+	ASSERT(srcs   != NULL);
```

cppcheck

[error] Common realloc mistake

(memleakOnRealloc)